From 924208952dbd8408b20b9357d7154f7aa5081b3e Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Sat, 17 Nov 2012 04:36:58 +0200
Subject: [PATCH 12/22] Fix for localizations of Detailed List View headers.

Detailed List View headers appeared untranslated, macro _() should be added.
---
 src/gtk/fm-folder-model.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gtk/fm-folder-model.c b/src/gtk/fm-folder-model.c
index 36f0d00..6072b3e 100644
--- a/src/gtk/fm-folder-model.c
+++ b/src/gtk/fm-folder-model.c
@@ -1783,7 +1783,7 @@ const char* fm_folder_model_col_get_title(FmFolderModel* model, FmFolderModelCol
     if(G_UNLIKELY(!FM_FOLDER_MODEL_COL_IS_VALID(col_id)
        || column_infos[col_id] == NULL)) /* invalid id */
         return NULL;
-    return column_infos[col_id]->title;
+    return _(column_infos[col_id]->title);
 }
 
 /**
-- 
1.8.0.1

